### ISSUE TITLE
Feature: add side-effect imports category for grouping

### ISSUE THREAD

**Issue by ai** [azat-io/eslint-plugin-perfectionist#4]
### What rule do you want to change?

sort-imports

### Describe the problem

Side effect imports like:

```js
import from './set-production-env.js'
```

should be above all others import. And fo-course there should not be auto-srto for these type of imports.

Can we add another category to grouping? If we already support it (and have no option to configure, which is fine) let’s add an example to docs.

### Code example

```js
import from './set-production-env.js'
```

### Additional comments

Maybe we need another logic for `.css` imports

### Validations

- [X] Read the [docs](https://eslint-plugin-perfectionist.azat.io).
- [X] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by azat-io** [comment]
Sounds reasonable. We currently have 13 types of imports that can be grouped together.

```ts
type Group =
  | 'builtin'
  | 'external'
  | 'internal'
  | 'parent'
  | 'sibling'
  | 'index'
  | 'object'
  | 'type'
  | 'internal-type'
  | 'parent-type'
  | 'sibling-type'
  | 'index-type'
  | 'unknown'
```

In the current implementation, side-effect imports will be assigned to the `'siblings'` or `'parent'` type. Perhaps a new `'side-effect'` group should be added. And maybe group `'side-effect-css'`.

Do you have any other ideas for what the API might look like?

**Comment by ai** [comment]
I like new category `'side-effect'` and `'css'`

**Comment by azat-io** [comment]
`'side-effect'` group added in 02f51fb38cc3c3140b2d6f6e957191966db41fd4
`'style'` group added in 05bf0f7ee6bf4aa633c1d9d1ed403a32d0b46b36 (not `'css'`, because it supports `.sass`, `.less`, `.scss`, `.pcss`, `.sss` and `.css` extensions)

Released in [v1.1.0](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v1.1.0)
